src/sql/workbench/api/common/extHostModelViewDialog.ts

```diff
@@ -888,7 +888,10 @@ export class ExtHostModelViewDialog implements ExtHostModelViewDialogShape {
 	public updateWizardPage(page: azdata.window.WizardPage): Thenable<void> {
 		let handle = this.getHandle(page);
 		if (page.customButtons) {
-			page.customButtons.forEach(button => this.updateButton(button));
+			page.customButtons.forEach(button => {
+				button.secondary = true;
```

Comment:
What about letting the callers control this if they want - and just default to secondary if it's undefined?

Reply:
a default value (false) is already set when the button is created, for the context of dialog/wizard, the primary button is already set to next/done button, we would want to make the custom buttons secondary.